### ISSUE TITLE
Not showing SVGs in Firefox on Desktop and Mobile

### ISSUE THREAD

**Issue by ch3ckmat3** [peiffer-innovations/websafe_svg#7]
Does anyone having this issue? I am using v1.1.0 of the plugin and it works flawlessly everywhere until I discovered that it is not showing any SVGs on any flavour of Firefox browser on my Windows 10 as well as on my Android.

Am I missing something?

**Comment by jpeiffer** [comment]
Flutter did a major release recently that helped with other web issues.  Does this still happen after upgrading to the stable 1.20.0?  If so, can you please provide an example where the issue occurs?

**Comment by oandreyev** [comment]
I can confirm this issue. An example in https://events.racetime.app/#/signin
The small round button should contain Google logo but it does not appear on FF. Chrome and Safari are good.
The Flutter version is 1.20.1

**Comment by fischerscode** [comment]
I have the same problem,
it seems like the height and with is not set in Firefox.
When setting it manually (inspect elements) the SVG appears.

**Comment by fischerscode** [comment]
I think this is a flutter issue.
This image is visible in chrome and invisible in firefox.
```dart
Image.network(
"data:image/svg+xml;base64,PHN2ZyByb2xlPSJpbWciIHZpZXdCb3g9IjAgMCAyNCAyNCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj48dGl0bGU+QWJzdHJhY3QgaWNvbjwvdGl0bGU+PHBhdGggZD0iTTEyIDBjOS42MDEgMCAxMiAyLjM5OSAxMiAxMiAwIDkuNjAxLTIuMzk5IDEyLTEyIDEyLTkuNjAxIDAtMTItMi4zOTktMTItMTJDMCAyLjM5OSAyLjM5OSAwIDEyIDB6bS0xLjk2OSAxOC41NjRjMi41MjQuMDAzIDQuNjA0LTIuMDcgNC42MDktNC41OTUgMC0yLjUyMS0yLjA3NC00LjU5NS00LjU5NS00LjU5NVM1LjQ1IDExLjQ0OSA1LjQ1IDEzLjk2OWMwIDIuNTE2IDIuMDY1IDQuNTg4IDQuNTgxIDQuNTk1em04LjM0NC0uMTg5VjUuNjI1SDUuNjI1djIuMjQ3aDEwLjQ5OHYxMC41MDNoMi4yNTJ6bS04LjM0NC02Ljc0OGEyLjM0MyAyLjM0MyAwIDExLS4wMDIgNC42ODYgMi4zNDMgMi4zNDMgMCAwMS4wMDItNC42ODZ6Ii8+PC9zdmc+",
height: 70,
width: 70,
)
```

**Comment by muhleder** [comment]
You can work around this by setting an explicit width and height in the svg itself.

```
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" width="24" height="24">

....
</svg>
```

Also, specifiying `fit: BoxFit.fill` will allow the svg to resize to the dimensions specified in dart rather than being locked to the width and height in the svg.

**Comment by jpeiffer** [comment]
I saw the original Flutter issue got closed, but agree the issue is either there or potentially Firefox itself.

Would you like me to open a new, clean, issue against Flutter?  Or did a new one already get created?

**Comment by muhleder** [comment]
@jpeiffer I haven't got around to creating a new issue yet, might get more attention from you since you're the maintainer of websafe_svg

**Comment by jpeiffer** [comment]
Heh, I doubt it.  I tried twice to fix a long standing bug in Flutter's Firebase plugin, with PR's, and they ignored me...  But I'll give it a shot and try to get one up by the end of the week.

**Comment by jpeiffer** [comment]
Fingers crossed...
https://github.com/flutter/flutter/issues/66412

**Comment by jpeiffer** [comment]
It looks like this has landed in the 1.25 milestone for Flutter and is fixed.  I'm closing the issue here as well.
